lib/Request.js

```diff
@@ -188,6 +188,14 @@ Request.prototype.validateQuery = function(schema, defaults) {
                 value = parseFloat(value);
                 if (isNaN(value)) return typeError(key);
                 break;
+
+            case 'boolean':
```

Comment:
would you mind please seperating this into it's own if statement ... that way: if (schema.properties.indexOf('boolean') !== -1) {...} ... I have another PR to remove this switch statement to allow validation and casting of a schema object having multiple types such as: type: ['integer', null] . If you move into it's own if statement it will not create as much merge conflicts for my PR ... thanks!